Master

reptalex opened the pull request:
Updated PCM and PVA tutorials

PCM - includes updated language

PVA - includes edgePCA-PhyloFactor comparison

reptalex committed:
Updated tutorials

Modified language in PCM tutorial and added edgePCA-PhyloFactor comparison to PVA tutorial.

reptalex committed:
Updated PCM and PVA tutorials

PCM tutorials included updated language

PVA tutorials include edgePCA-PhyloFactor comparison

mortonjt commented:
Looks pretty solid - I'll still need to generate the html pages.  Do you feel that this is good to go?

reptalex commented:
I think so. I'll take a close look over all of them before Monday and get back to ya

Sent from miPhone


> On Apr 22, 2017, at 7:14 PM, Jamie Morton <notifications@github.com> wrote:
> 
> Looks pretty solid - I'll still need to generate the html pages. Do you feel that this is good to go?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

reptalex committed:
Revised kernel methods tutorial

Includes fixed typos and summary on assumptions, limitations, and
special considerations of kernel methods.

reptalex committed:
Revised kernel methods tutorial

Includes fixed typos and summary on assumptions, limitations, and
special considerations of kernel methods.

reptalex commented:
I looked over the kernel methods and made a few revisions to the language (no changes to the code). It looks great, so I mainly integrated the language with other tutorials and the paper itself. The main revisions were:

-Removal of indirect pronouns (such as "This" and "Those" and "These" at the beginning of sentences - makes for lower RAM usage in the reader as they don't have remember the subject being referred to :-D)

-Removal of "high-level" in favor of "community-level" understanding ("high" implies an axis of understanding along which "level" is ranked from "low" to "high", and it's unclear to me which axis that is).

-Addition of a set of final, synthesis paragraphs which address the considerations of novel kernel methods, including the nature of the data (compositional or not?), the treatment of phylogeny (branch lengths re-scaled?), and the underlying evolutionary and ecological model.

mortonjt commented on tutorials/kernel_methods.md:
```diff
@@ -1,19 +1,19 @@
+Unifrac attempts to resolve the differences between microbial communities, even when species do not overlap, by incorporating the extent of shared ancestry as a measure of similarity or, conversely, the extent of novel evolutionary lineages, optionally weighted by the relative abundance of novel lineages, as a measure of dissimliarity. Unifrac has been shown to be a variant of the [Earth Mover's  ...
 
-To show how to run Unifrac, let's first create the familar RNA copy number simulation shown in the PVA tutorial.  We'll first want to boot up rpy2 to embed R inside of Python.  We want to do this, since we will be using the scikit-bio implementation of Weighted Unifrac.
+To show how to run Unifrac, we'll use hte RNA copy number simulation shown in the PVA tutorial.  We'll boot up rpy2 to embed R inside of Python in order to use the scikit-bio implementation of Weighted Unifrac.
```
hte -> the

mortonjt commented:
Awesome!!

On Sun, Apr 23, 2017 at 11:11 AM, Alex Washburne <notifications@github.com>
wrote:

> I looked over the kernel methods and made a few revisions to the language
> (no changes to the code). It looks great, so I mainly integrated the
> language with other tutorials and the paper itself. The main revisions were:
>
> -Removal of indirect pronouns (such as "This" and "Those" and "These" at
> the beginning of sentences - makes for lower RAM usage in the reader as
> they don't have remember the subject being referred to :-D)
>
> -Removal of "high-level" in favor of "community-level" understanding
> ("high" implies an axis of understanding along which "level" is ranked from
> "low" to "high", and it's unclear to me which axis that is).
>
> -Addition of a set of final, synthesis paragraphs which address the
> considerations of novel kernel methods, including the nature of the data
> (compositional or not?), the treatment of phylogeny (branch lengths
> re-scaled?), and the underlying evolutionary and ecological model.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/knightlab-analyses/phylogenetic-tutorials/pull/8#issuecomment-296470323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD_a3TwT-Sy9IZPz8qqZ4LETEL3xskLsks5ry5Q8gaJpZM4NFOt3>
> .
>

reptalex commented:
Good catch!

On Sun, Apr 23, 2017 at 12:12 PM, Jamie Morton <notifications@github.com>
wrote:

> Awesome!!
>
> On Sun, Apr 23, 2017 at 11:11 AM, Alex Washburne <notifications@github.com
> >
> wrote:
>
> > I looked over the kernel methods and made a few revisions to the language
> > (no changes to the code). It looks great, so I mainly integrated the
> > language with other tutorials and the paper itself. The main revisions
> were:
> >
> > -Removal of indirect pronouns (such as "This" and "Those" and "These" at
> > the beginning of sentences - makes for lower RAM usage in the reader as
> > they don't have remember the subject being referred to :-D)
> >
> > -Removal of "high-level" in favor of "community-level" understanding
> > ("high" implies an axis of understanding along which "level" is ranked
> from
> > "low" to "high", and it's unclear to me which axis that is).
> >
> > -Addition of a set of final, synthesis paragraphs which address the
> > considerations of novel kernel methods, including the nature of the data
> > (compositional or not?), the treatment of phylogeny (branch lengths
> > re-scaled?), and the underlying evolutionary and ecological model.
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/knightlab-analyses/phylogenetic-
> tutorials/pull/8#issuecomment-296470323>,
> > or mute the thread
> > <https://github.com/notifications/unsubscribe-auth/AD_a3TwT-
> Sy9IZPz8qqZ4LETEL3xskLsks5ry5Q8gaJpZM4NFOt3>
> > .
>
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/knightlab-analyses/phylogenetic-tutorials/pull/8#issuecomment-296470572>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOGtg1mJjmqggRX1LrceFivHWG4eXQBPks5ry5SSgaJpZM4NFOt3>
> .
>

mortonjt commented on tutorials/PVA_tutorial.Rmd:
````diff
@@ -152,21 +152,29 @@ Evolutionary leaps occur along edges in the phylogeny and can cause geometric ch
+Barring null simulation or future research justifying null distributions of objective statistics, phylofactorization should be treated as an exploratory technique. In a commmunity with `nspecies` OTUs, one can obtain up to `nspecies-1` factors, though the significance and evolutionary interpretation of downstream factors will be questionable. Washburne et al. (2017) attempted to define a criterio ...
+```{r Phylofactorization, warning=FALSE}
+# How to obtain 4 factors with optional parallelization:
+# PhyloFactor(Data=OTUTable,tree,X=disturbance_frequency,nfactors=4,ncores=2)
````
Should this be removed?

reptalex replied:
Do you mean the final comments, or the whole paragraph + final comments?

mortonjt replied:
Just the final comment it just seemed redundant that's all.

mortonjt commented on tutorials/PVA_tutorial.Rmd:
```diff
@@ -152,21 +152,29 @@ Evolutionary leaps occur along edges in the phylogeny and can cause geometric ch
-The first factor separates a 4-member Monophyletic clade from a 6-member "monophyletic" clade. Both clades are monophyletic because phylofactorization considers an unrooted tree, but the researcher can make further assumptions regarding monophyly. 
+The stopping criterion halted phylofactorization at the third factor. The first factor separates a 4-member Monophyletic clade from a 6-member "monophyletic" clade. Both clades are monophyletic because phylofactorization considers an unrooted tree, but the researcher can make further assumptions regarding monophyly. 
+
+The regression model from the first factor explains 32% of the total variance in the dataset. The F-statistics from regresion are displayed along with P-values from F-tests. The P-values from phylofactorization are based on an F-test and do not correct for multiple comparisons. The likelihood of seeing an explained variance as or more extreme than the one observed, given a phylofactorization of a ...
```
regresion -> regression

mortonjt commented:
Just one minor comment.  Once that is fixed, I can merge this in and update the website!
